### PR TITLE
Reference consumer test

### PR DESCRIPTION
**Pull request by d-gregorczyk**

### CODE REVIEW

**Comment by klotzt-draeger** [comment]
can we move the code to subfolder called "sdc11073"

File: Tooling/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+sdc11073
+unittest-xml-reporting
```
Comment (klotzt-draeger) [inline Tooling/requirements.txt:2]: i think we need to fix the cryptography version, i got issues when installing without a fixed version.

Comment (klotzt-draeger) [reply Tooling/requirements.txt:2]: nevermind, with a new pip version it's all good.

**Comment by ToddCooper** [comment]
We will review on the SDPi Friday call in 30 and complete the pull then.

Thanks!

~>Todd 탇

*Soli Deo gloria*

M:  +1 858.442.9200
S:  ToddCooperAFC

**Comment by d-gregorczyk** [comment]
I cannot really assess if that code does what it is supposed to do. If Tobias is generally fine, I would merge the commit. Please let me know.

**Pull request merged**